.travis.yml

```diff
@@ -12,18 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-language: node_js
+dist: trusty
+language: minimal
 
-node_js:
-  - 10
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - wget
+      - pkg-config
 
 cache:
   yarn: true
```

Comment:
Does this make any difference?

Reply:
I imagine it still matters. When bazel runs yarn it's nearly identical to how a user would run it